server/app/uk/gov/ons/addressIndex/server/modules/SystemBootstrap.scala

```diff
@@ -0,0 +1,15 @@
+package uk.gov.ons.addressIndex.server.modules
+
+import java.io.File
+import javax.inject.{Inject, Singleton}
+import play.api.Logger
+
+@Singleton
+class SystemBootstrap @Inject()() {
+  System.load(
+    new File(
+      "parsers/src/main/resources/libbackend.so"
```

Comment:
still up in the air about this one, not too sure, maybe a discussion with Al once this is running